### DISCUSSION TITLE
`win + shfit + s` not work when using `LSHF`

### DISCUSSION THREAD

**Discussion by u-Sir** [cajhin/capsicain#78] [Q&A]
`win + LSHF + s` not work, in debug mod can't find `s` key info, but `win + RSHF + s` works, and all keys detected. how to fixed it?
my config:
```
[CONFIG_1]
#OPTION debug
OPTION configName one

REWIRE LWIN CAPS
REWIRE CAPS LWIN
COMBO CAPS [] > sequence(CAPS_CONFIGSWITCH:2)


[CONFIG_2]
OPTION ConfigName two

REWIRE LWIN CAPS
REWIRE CAPS LWIN

REWIRE 1 F1
REWIRE 2 F2
REWIRE 3 F3
REWIRE 4 F4
REWIRE 5 F5
REWIRE 6 F6
REWIRE 7 F7
REWIRE 8 F8
REWIRE 9 F9
REWIRE 0 F10
REWIRE - F11
REWIRE = F12

COMBO CAPS [] > sequence(CAPS_CONFIGSWITCH:1)
```

**Comment by cajhin** [comment]
I don't understand what you want to happen. You Rewire LWIN to Caps, so when you press Shift+Lwin+s you would get Shift+Caps+S ?
Except that with COMBO CAPS [], as soon as you press Lwin (with any modifier), you switch between config 1 and 2.

I guess you want to have the number keys act as function keys, and be able to toggle that on/off with the caps key?
Do you actually want to move around your LWin and Caps?

Because your config_1 does not do anything, you could also turn capsicain on/off instead.
See GLOBAL capsicainOnOffKey in [https://github.com/cajhin/capsicain/wiki/Keyword%3A-GLOBAL]

In the standard config, I use combo Tab+1 for F1

**Reply by u-Sir** [reply]
> I don't understand what you want to happen. You Rewire LWIN to Caps, so when you press Shift+Lwin+s you would get Shift+Caps+S ? Except that with COMBO CAPS [], as soon as you press Lwin (with any modifier), you switch between config 1 and 2.
> 
> I guess you want to have the number keys act as function keys, and be able to toggle that on/off with the caps key? Do you actually want to move around your LWin and Caps?
> 
> Because your config_1 does not do anything, you could also turn capsicain on/off instead. See GLOBAL capsicainOnOffKey in [https://github.com/cajhin/capsicain/wiki/Keyword%3A-GLOBAL]
> 
> In the standard config, I use combo Tab+1 for F1

ok, what happened after `capslock` <-> `lwin`：
screen capture
- `lwin(caps) + left shift + s`  not work, how can i make it work?
- `lwin(caps)  + right shift + s` work
simplified config:
```
[CONFIG_1]
#OPTION debug
OPTION configName one

REWIRE LWIN CAPS
REWIRE CAPS LWIN
```
